hivemind/p2p/p2p_daemon_bindings/datastructures.py

```diff
@@ -0,0 +1,186 @@
+import hashlib
+from typing import Union, List, Sequence, Any
+
+import base58
+import multihash
+
+from multiaddr import Multiaddr, protocols
+from hivemind.proto import p2pd_pb2
+
+from hivemind.p2p.p2p_daemon_bindings.keys import PublicKey
```

Comment:
```suggestion
import multihash
from multiaddr import Multiaddr, protocols

from hivemind.p2p.p2p_daemon_bindings.keys import PublicKey
from hivemind.proto import p2pd_pb2
```

nit: I suggest to sort imports and divide it to stdlib/third-party/project-wide groups.